photo-editor/dom.js

```diff
@@ -0,0 +1,12 @@
+function editPhoto() {
+    const filters = document.querySelectorAll('input');
+    
+    filters.forEach(filter => filter.addEventListener('change', filtersUpdate));
```

Comment:
it's a bad practice to use functions before its declaration. try to change the functional declaration to arrow function and check you console